src/views/Home.jsx

```diff
@@ -3,6 +3,8 @@ import { SingleList } from '../components/SingleList.jsx';
 import { useState } from 'react';
 import { createList } from '../api/firebase.js';
 import { useNavigate } from 'react-router-dom';
+import { auth } from '../api/config.js';
+import { IoBulbOutline } from 'react-icons/io5';
```

Comment:
praise: great use of that library we imported - i honestly forgot we had so much variety to draw from